docs/shared/tutorials/npm-workspaces.md

```diff
@@ -386,44 +386,54 @@ This tutorial walked you through how Nx can improve the local development experi
 - Nx Agents [efficiently distribute tasks across machines](/ci/concepts/parallelization-distribution) ensuring constant CI time regardless of the repository size. The right number of machines is allocated for each PR to ensure good performance without wasting compute.
 - Nx Atomizer [automatically splits](/ci/features/split-e2e-tasks) large e2e tests to distribute them across machines. Nx can also automatically [identify and rerun flaky e2e tests](/ci/features/flaky-tasks).
 
-### Generate a CI Workflow
+### Connect to Nx Cloud
 
-If you are starting a new project, you can use the following command to generate a CI workflow file.
+Nx Cloud is a companion app for your CI system that provides remote caching, task distribution, e2e tests deflaking, better DX and more.
 
-```shell
-npx nx generate ci-workflow --ci=github
-```
+Now that we're working on the CI pipeline, it is important for your changes to be pushed to a GitHub repository.
 
-{% callout type="note" title="Choose your CI provider" %}
-You can choose `github`, `circleci`, `azure`, `bitbucket-pipelines`, or `gitlab` for the `ci` flag.
-{% /callout %}
+1. Commit your existing changes with `git add . && git commit -am "updates"`
+2. [Create a new GitHub repository](https://github.com/new)
+3. Follow GitHub's instructions to push your existing code to the repository
 
-This generator creates a `.github/workflows/ci.yml` file that contains a CI pipeline that will run the `lint`, `test`, `build` and `e2e` tasks for projects that are affected by any given PR.
+You can verify that the GitHub repository is set up correctly by running the following command:
 
-The key line in the CI pipeline is:
+```shell
+git remote get-url origin
+```
 
-```yml
-- run: npx nx affected -t lint test build
+Now connect your repository to Nx Cloud with the following command:
+
+```shell
+npx nx connect
 ```
 
-### Connect to Nx Cloud
+Once the script is finished, it will print a link in the terminal to register your repository in your [Nx Cloud](https://cloud.nx.app) account. Click the link and follow the steps provided to make sure Nx Cloud is enabled on the main branch of your repository.
 
-Nx Cloud is a companion app for your CI system that provides remote caching, task distribution, e2e tests deflaking, better DX and more.
+When you're finished, you'll need to pull down the changes that were merged into the hosted repository with this command:
```

Comment:
Could we add more guidance around here? Like what it actually looks like, some screenshots etc? We'd need that also on the CI tutorials anyway, see comment here:
https://github.com/nrwl/nx/pull/26846